product_variant_sale_price/models/product_product.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def _update_fix_price(self, vals):
+        if "list_price" in vals:
+            self.mapped("product_variant_ids").write({"fix_price": vals["list_price"]})
+
+    @api.model_create_multi
+    def create(self, vals):
+        records = super().create(vals)
+        for i, product_tmpl in enumerate(records):
+            single_vals = vals[i] if isinstance(vals, list) else vals
+            product_tmpl._update_fix_price(single_vals)
+        return records
+
+    def write(self, vals):
+        res = super().write(vals)
+        if self.env.context.get("skip_update_fix_price", False):
+            return res
+        for template in self:
+            template._update_fix_price(vals)
+        return res
+
+
```

Comment:
On the other hand, within the sale module it would be advisable to hide the badge that is generated when the _price_extra_ is informed.

https://github.com/odoo/odoo/blob/5ac58ebf983c1c02019253c0430fde3502e13e7c/addons/sale/views/variant_templates.xml#L119C1-L142

For this reason and to continue with the structure of our changes, I propose modifying the __get_combination_info()_ method so that it always reports that the `price_extra = 0.0`

```suggestion
    def _get_combination_info(self, combination=False, product_id=False, add_qty=1, pricelist=False, parent_combination=False, only_template=False):
        res = super(ProductTemplate, self)._get_combination_info(combination, product_id, add_qty, pricelist, parent_combination, only_template)
        res['price_extra'] = 0.0
        return res
```

I await your feedback to know what you think about this!! :D

Reply:
LGTM!

Reply:
Thank you for the very good suggestions. I applied the suggested changes.